### PR TITLE
Add a link to open the tweet in browser

### PR DESCRIPTION
**Pull request by Jebzou**

### CODE REVIEW

**Comment by Jebzou** [comment]
Maybe a button would be easier to use

**Comment by myangelkamikaze** [comment]
Hmm, it might be better to just add a [hyperlink](https://learn.microsoft.com/en-us/dotnet/api/system.windows.documents.hyperlink?view=windowsdesktop-7.0) at the bottom of the tweet or top or wherever. Double click on the tweet itself is not very discoverable yeah. `VersionInformation.xaml` has that for example (it's not mvvm there, your current implementation is better).

File: ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs [hunk 1]
```diff
@@ -19,4 +22,22 @@ public class TweetViewModel : ObservableObject
 	public string? Link { get; set; }
 
 	public string? Author { get; set; }
+
+	[RelayCommand]
+	private void OpenTweet()
+	{
+		try
+		{
+			ProcessStartInfo psi = new ProcessStartInfo
```
Comment (myangelkamikaze) [inline ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs:31]: Target typed new.

File: ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs [hunk 2]
```diff
@@ -19,4 +22,22 @@ public class TweetViewModel : ObservableObject
...
+			{
+				FileName = Link,
+				UseShellExecute = true
```
Comment (myangelkamikaze) [inline ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs:34]: Trailing comma.

File: ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs (hunk 1 above)
Comment (Jebzou) [reply ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs]: method has been removed

File: ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs (hunk 2 above)
Comment (Jebzou) [reply ElectronicObserver/Window/Tools/Telegram/TweetViewModel.cs]: method has been removed

**Pull request merged**